gdal/frmts/jpegxr/frmt_jpegxr.html

```diff
@@ -0,0 +1,80 @@
+<html>
+<head>
+<title>JPEGXR -- JPEG XR File Format</title>
+</head>
+
+<body bgcolor="#ffffff">
+
+<h1>JPEGXR -- JPEG XR File Format</h1>
+
+<p>The JPEG XR format is supported for reading, and batch writing, but not
+update in place.  JPEG XR files are represented as one band (Greyscale) or three
+band (RGB) or four band (RGB with Alpha) datasets with Byte valued bands.<p>
+
+<p>The driver is implemented based on the 
+    <a href="https://jxrlib.codeplex.com/">jxrlib</a> - open source (BSD)
+    implementation of JPEG XR format by Microsoft.</p>
+
+<h2>Driver Features</h2>
+
+<ul>
+<li>Compression - lossless by default, controlled with <code>QUALITY</code> option.</li>
+<li>Image Structure - entire image is decoded; tiling not yet supported.</li>
+<li>Channel Organization - Grey, RGB, BGR, RGBA, BGRA.</li>
+<li>Color - RBG images YUV with 8-bit per sample and
+<a href="https://en.wikipedia.org/wiki/Chroma_subsampling">4:4:4</a> full chroma
+resolution encoding by default.</li>
+</ul>
+
+
+<p>The driver preserves full chroma resolution 
+
+<h2>Creation Options</h2>
+
+<p>JPEG XR files are created using the "JPEG" driver code.
+ JPEG file creation is implemented by the batch (<code>CreateCopy</code>) method. </p>
```

Comment:
@schwehr Again, no reason. Please, point me to the examplar of valid HTML formatting in GDAL and I will reformat `frmt_jpegxr.html` at once.